service/Makefile.am

```diff
@@ -79,6 +79,7 @@ EXTRA_DIST = .gitignore \
              debian/rules \
              dox/blurb.md \
              dox/Doxyfile.in \
+             fuzz_test/run_fuzz_tests.sh \
```

Comment:
since we have a fuzz_test/Makefile.mk maybe add

```
EXTRA_DIST += fuzz_test/run_fuzz_test.sh
```

to fuzz_test/Makefile.mk?